include/compose.bash

```diff
@@ -677,6 +677,11 @@ EOF
             - VAULT_PORT=$VAULT_BIND_PORT
             - VAULT_ROOT_TOKEN=$VAULT_ROOT_TOKEN
             - "CAAS_URL=$CAAS_URL"
+            - "SHOW_TERMINATED_CLUSTERS_ACTIVE=$CB_SHOW_TERMINATED_CLUSTERS_ACTIVE"
```

Comment:
https://github.com/hortonworks/cloudbreak/blob/1c719d7edab99af3b7a0749b7e3e0c630e32dcfc/core/src/main/java/com/sequenceiq/cloudbreak/service/stack/ShowTerminatedClusterConfigService.java#L32 Cloudbreak seeking for `cb.show.terminated.clusters.active` will never find this variable.

Reply:
You are right ... Fixed